activerecord/lib/active_record/migration.rb

```diff
@@ -555,6 +555,10 @@ def call(env)
 
       private
 
+        def rails_migration_path?(env)
+          Rails.env.development? && env["ORIGINAL_FULLPATH"] == '/rails/migrate'
```

Comment:
Is this middleware even included in environments other than development or can we ditch the check? (It reads somewhat weird we're doing an environment check in a method that's about being on some path.)